client/components/ui/TimePicker.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="relative">
+  <div type="text" readonly tabindex="0" @focus="focusDigit('second0')" class="relative">
```

Comment:
```suggestion
  <div tabindex="0" @focus="focusDigit('second0')" class="relative">
```
The `type` and `readonly` attributes aren't available on div elements. Unless there is some hacky thing I'm not aware of? Let me know if I'm missing something here

Reply:
It's a hacky thing. I was just trying to figure out how to make the keyboard show up on mobile.

Reply:
Hacky but does it work? Afaik those attributes aren't supported on div